sonar-plugin-api/src/main/java/org/sonar/api/profiles/ProfileDefinition.java

```diff
@@ -21,16 +21,19 @@
 
 import org.sonar.api.ExtensionPoint;
 import org.sonar.api.server.ServerSide;
+import org.sonar.api.server.profile.BuiltInQualityProfilesDefinition;
 import org.sonar.api.utils.ValidationMessages;
 
 /**
  * Define a profile which is automatically registered during sonar startup.
  * The components <code>AnnotationProfileParser</code> and <code>XMLProfileParser</code> can be used to help implementing the method create().
  *
  * @since 2.3
+ * @deprecated since 6.6 use {@link BuiltInQualityProfilesDefinition}
  */
 @ServerSide
 @ExtensionPoint
+@Deprecated
 public abstract class ProfileDefinition {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)